graphene_django_cud/mutations/core.py

```diff
@@ -279,8 +280,10 @@ def create_obj(
         foreign_key_extras_field_names = get_fk_all_extras_field_names(foreign_key_extras)
 
         for field_name, context_name in auto_context_fields.items():
-            if hasattr(info.context, context_name):
-                model_field_values[field_name] = getattr(info.context, context_name)
+            try:
+                model_field_values[field_name] = attrgetter(context_name)(info.context)
+            except:
+                pass
```

Comment:
Which exceptions are likely to arise here? 

I'm not overly familiar with the `attrgetter` function. How exactly does it work (and how does it differ from getattr)?